Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM gcr.io/google_appengine/base
 # at https://hub.docker.com/r/google/cloud-sdk/~/dockerfile/
 
 ENV \
-	CACHE_BUST=1 \
+	CACHE_BUST=2 \
```

Comment:
Okay, cool. Just as long as there's a plan to include the version number at some point :)